CMakeLists.txt

```diff
@@ -206,7 +236,7 @@ slang_add_target(
     STATIC
     EXCLUDE_FROM_ALL
     USE_EXTRA_WARNINGS
-    LINK_WITH_PRIVATE miniz lz4_static Threads::Threads ${CMAKE_DL_LIBS}
+	LINK_WITH_PRIVATE ${SLANG_CORE_LIBRARIES}
```

Comment:
indentation, sorry. This file roughly follows what `gersemi` formats it as, although that's not enforced